ros2_ws/src/vision/path_detection/path_detection/obstacles.py

```diff
@@ -38,9 +36,9 @@ def check_collision(a, b, c, x, y, radius):
     if radius < dist:
         return 0, False
     else:
+        # need to know what these consts are for (need to be update?? 1/28/23)
```

Comment:
it'll be a good idea to figure out what this is for before you start testing (and update it if necessary). Let me know if you want any help with that.